Documentation/tls/tls-ingress.md

```diff
@@ -0,0 +1,58 @@
+# Enabling custom Tectonic Console TLS certificates
+
+Use Tectonic's [Ingress TLS module][ingress-module] to enable user provided Ingress certificates.
+
+The module does not contain any logic, but just passes user provided certificates from its input directly to its output. This is to prevent changing existing references to the `tls/ingress/self-signed` module, hence all `tls/ingress/*` modules share the same outputs.
+
+## Usage
+
+Comment out the existing self-signed ingress TLS in your platform, i.e. `platforms/aws/tectonic.tf`:
+
+```
+/*
+module "ingress_certs" {
+  source = "../../modules/tls/ingress/self-signed"
+
+  base_address = "${module.dns.ingress_internal_fqdn}"
+  ca_cert_pem  = "${module.kube_certs.ca_cert_pem}"
+  ca_key_alg   = "${module.kube_certs.ca_key_alg}"
+  ca_key_pem   = "${module.kube_certs.ca_key_pem}"
+}
+*/
+```
+
+Configure the user provided certificate paths in your platform, i.e. `platforms/aws/tectonic.tf`:
+
+```
+module "ingress_certs" {
+  source = "../../modules/tls/ingress/user-provided"
+
+  ca_cert_pem_path = "/path/to/ca.crt"
+  cert_pem_path    = "/path/to/ingress.crt"
+  key_pem_path     = "/path/to/ingress.key"
+}
+```
+
+The signed ingress certificate must have the following Subject Alternative Name (SAN) and Key Usage associations:
```

Comment:
Why is the SAN Needed? 
https://www.chromestatus.com/feature/4981025180483584

All browsers have made the shift at this point. Make a point of this. Folks are still acquiring certs without the SAN correctly set. We should use this opportunity to educate them. Super Super important that the SAN is set.